client/settings/phone-input/style.scss

```diff
@@ -6,7 +6,21 @@
 	display: flex;
 	flex-direction: column;
 	gap: $gap;
-	margin-top: $gap-large;
```

Comment:
This margin was applied even when the "save my info" container didn't have any content.